docs/api-src/audpsychometric.rst

```diff
@@ -3,84 +3,17 @@ audpsychometric
 
 .. automodule:: audpsychometric
 
-Library to facilitate evaluation and processing of annotated speech.
-
-Pychometric Analysis
---------------------
-
-.. autosummary::
-    :toctree:
-    :nosignatures:
-
-    cronbachs_alpha
-    congeneric_reliability
-    intra_class_correlation
-
-The module currently contains two reliability coefficients
-from the family of structural equation model (SEM)-based
-reliability coefficients.
-One of them is Cronbach's alphas
-in the function :func:`audpsychometric.cronbachs_alpha`.
-This classical coefficient assumes *tau equivalence*
-which requires factor loadings to be homogeneous.
-The second coefficient
-in the function :func:`audpsychometric.congeneric_reliability`
-relaxes this assumption
-and only assumes a `one-dimensional congeneric reliability`_ model:
-congeneric measurement models are characterized by the fact
-that the factor loadings of the indicators
-do not have to be homogeneous,
-i.e. they can differ.
-
-In addition,
-the module implements *Intraclass Correlation (ICC)* analysis.
-ICC is based on the analysis of variance of a class of coefficients
-that are based on ANOVA
-with ratings as the dependent variable,
-and terms for targets
-(like e.g rated audio chunks),
-raters and their interaction are estimated.
-Different flavors of ICC are then computed
-based on these sum of squares terms.
-
-Note that the CCC_ is conceptually and numerically related to the ICC.
-We do not implement it here,
-as there are other implementations available,
-e.g. :func:`audmetric.concordance_cc`.
-
-
-Gold Standard Calculation
--------------------------
-
 .. autosummary::
     :toctree:
     :nosignatures:
 
     agreement_categorical
```

Comment:
There is another repercussion of renaming the functions:  I know of people that have used the gold standard functions under their old names.  So I wonder whether there be a deprecation warning?

Reply:
There was never a release of this package, so there will be no deprecation warning.

Reply:
Ah then they used editable installations. So of course then not needed. 
Resolving.